schemas/common/phone-interaction.schema.json

```diff
@@ -0,0 +1,79 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/phoneinteraction",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Phone Interaction",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Captures information in the context of a phone call conversation ",
+  "definitions": {
+    "phoneinteraction": {
+      "properties": {
+        "xdm:ID": {
+          "title": "ID of the call conversation",
+          "type": "string",
+          "description": "A unique identifier to represent the phone call conversation"
+        },
+        "xdm:fromWhoID": {
+          "title": "Initiator of the call",
+          "type": "string",
+          "description": "Person ID who initiated the call of the conversation"
+        },
+        "xdm:toWhoID": {
+          "title": "Call reciever",
+          "type": "string",
+          "description": "Person ID who recieved the call"
+        },
+        "xdm:disposition": {
+          "title": "Label the outcome of the phone call",
+          "type": "string",
+          "description": "The outcome of the call as next triggering actions such as 'demo scheduled','left voicemail', etc."
+        },
+        "xdm:duration": {
+          "title": "Time Spent during the call",
+          "type": "number",
+          "description": "Record the length of the phone conversation in seconds"
+        },
+        "xdm:type": {
+          "title": "Type of the phone call",
+          "type": "string",
+          "description": "A way to represent the nature of the call",
+          "meta:enum": {
+            "internal": "Internal",
+            "inbound": "Inbound",
+            "outbound": "Outbound"
+          }
+        },
+        "xdm:fromNumber": {
```

Comment:
As we used in the other B2B schemas, should we use the datatype here ?
"https://ns.adobe.com/xdm/context/phonenumber"